gcn_classic_to_json/notices/CALET_GBM_FLT_LC/__init__.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+
+from ... import utils
+
+
+def parse(bin):
+    if (bin[0] != 160) and (bin[39] != 10):
+        return
+
+    if np.all(bin[1:4]) and bin[12] and np.all(bin[13:16]) and np.all(bin[20:29]):
+        pass
```

Comment:
What is this supposed to do?

Reply:
@lpsinger  bin[0] is the packet_type which is 160 for calet and bin[39] is "\n". Since we have to use every field in the array, I'd figure I could use the data as a basic check.

The rest are either fields with no meaningful data i.e. for internal bookkeeping or empty fields. The code is just to pass the test that makes you use every field.

Reply:
> @lpsinger bin[0] is the packet_type which is 160 for calet and bin[39] is "\n". Since we have to use every field in the array, I'd figure I could use the data as a basic check.

See https://github.com/nasa-gcn/gcn-classic-to-json/pull/31.

Reply:
@lpsinger On bin 1-3:

- pkt_sernum is a number that starts at 1 and increments to infinity for every notice produced when the GCN program boots up but in practice it seems to be always 1 since the even updated alerts seem to be some time apart which resets the number. Here's an [example notice](https://gcn.gsfc.nasa.gov/other/216.milagro) from the MILARGO mission.
- pkt_hop_cnt from the documentation I believe was a count they used to verify that the packet went through all the required machines but became defunct after they changed their system.
- pkt_sod would typically correspond to alert_datetime, however the packets have no data regarding the date on which they were created. This conversion was possible in the text notices since both the date and time were available. I presume that creating the date for the text notices was handled internally.

For the rest, I'll make the corresponding changes

Reply:
> * pkt_sernum is a number that starts at 1 and increments to infinity for every notice produced when the GCN program boots up but in practice it seems to be always 1 since the even updated alerts seem to be some time apart which resets the number. Here's an [example notice](https://gcn.gsfc.nasa.gov/other/216.milagro) from the MILARGO mission.
> * pkt_hop_cnt from the documentation I believe was a count they used to verify that the packet went through all the required machines but became defunct after they changed their system.

Is this the case for all notice types? If so, then I suggest that we add code to ignore them to `__init__.py`.